backend/mora/service/employee.py

```diff
@@ -191,7 +219,10 @@ def get_one_employee(c, userid, user=None, full=False):
 
         r[mapping.ORG] = org.get_one_organisation(c, orgid)
         r[mapping.USER_KEY] = props['brugervendtnoegle']
-
+    elif details is EmployeeDetails.MINIMAL:
+        pass  # already done
+    elif details is EmployeeDetails.INTEGRATION:
+        r[mapping.INTEGRATION_DATA] = props.get("integrationsdata")
```

Comment:
We should parse the JSON here. Do we have tests in place for the case where someone injects invalid JSON into the attribute, or where it's unset?

Reply:
Maybe and no- we probably do not have that, But then again - it is not json that is injected - it is an object that is encoded using json.dumps - see above:

            if mapping.INTEGRATION_DATA in data:
                attrs['integrationsdata'] = json.dumps(
                    data[mapping.INTEGRATION_DATA]
                )

Reply:
> Maybe and no- we probably do not have that, But then again - it is not json that is injected - it is an object that is encoded using json.dumps - see above:

AFAICT, no such code exists within this function. `props` is assigned `user['attributter']['brugeregenskaber'][0]` in line 204, and not modified after that. In this particular statement, you're assigning the raw, unmodified value of the LoRA attribute — i.e. a string. You should assign the parsed JSON instead:
```suggestion
        r[mapping.INTEGRATION_DATA] = json.parse(props.get("integrationsdata", "null"))
```
…or something similar, depending on how you want to handle errors.